Client/Application/AppDelegate.swift

```diff
@@ -186,6 +188,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         self.profile = p
         return p
     }
+    
+    func updateShortcutItems(_ application: UIApplication) {
+        let newTabItem = UIMutableApplicationShortcutItem(type: "\(Bundle.main.bundleIdentifier ?? "").NewTab",
```

Comment:
Extract text values to `BraveStrings`, there's probably a text value for it already

Reply:
@iccub All the text items for this is in `InfoPlist.strings` localizations. I didn't want to change all of the code so I left it as is.

Should I copy them to `BraveStrings` file?

Reply:
oh right, you have to add them to `BraveStrings` then, all localized text files should be there

Reply:
Fixed.

Reply:
@iccub I am afraid moving existing strings to BraveStrings might break transifex. Since all the strings are also mapped to their locations, Brandon's change will delete the older translations and create new entries. Though we can get the translation back from the history, it is a time taking process. 
@Brandon-T I would suggest not adding translations like this as these will be overwritten by translations pulled from transifex.

If we really want to move it to BraveStrings, I would suggest creating another ticket for pulling translations from history.

Reply:
So what is the optimal solution here @danishjafri88?

Reply:
Well we can do either of these:
- Revert to previous state
or
- Let this pass, and create another issue to push these changes to transifex, transfer translations from history and pull again. Also remove the changes in strings file from this pr.

In long run, since these strings are used within code, they should be in BraveStrings.

Reply:
@iccub which one do you think is better. I would say go with second one.

Reply:
Fine by me, 
sorry for that @Brandon-T :D localization Strings work is tricky

Reply:
So do I revert the strings change? @iccub @danishjafri88